scene/realistic-light-and-shadows/settings.gradle

```
@@ -0,0 +1 @@
+rootProject.name = 'Realistic Lighting and Shadows'
```

We usually have this parameter as the name of the sample's folder. At the same time, we should probably rename the folder this sample sits in to reflect the full sample name `realistic-lighting-and-shadows`

```suggestion
rootProject.name = 'realistic-lighting-and-shadows'
```